app/decorators/ext_management_system_decorator.rb

```diff
@@ -10,4 +10,19 @@ def fonticon
   def fileicon
     "svg/vendor-#{image_name}.svg"
   end
+
+  def quadicon(_n = nil)
+    {
+      :top_left     => { :text => try(:hosts).try(:size) || 0 },
```

Comment:
isn't the ext_mgmt the top level? does it have hosts as an entity? or it's a default fallback?

Reply:
This is just deafult fallback.